resources/js/components/Search/Autocomplete.vue

```diff
@@ -28,24 +22,14 @@ export default {
 
     mounted() {
         this.$nextTick(() => {
-            this.$emit('mounted')
+            window.setTimeout(() => this.$emit('mounted'))
```

Comment:
What's this for? Could this also be another `$nextTick`? What if you move the `this.loaded` to outside of the parent `$nextTick`?

Reply:
Unfortunately the 2nd nextTick is still too early, i do think i can use requestAnimationFrame though.

Out of curiosity i set some performance marks
![image](https://github.com/user-attachments/assets/44d3bb8a-c44f-4a53-a018-06eab7f4aa7a)
You can see the small slivers at the top.

The first is the mounted method itself.
The second and third are $nextTicks nested in each other.
the fourth is the requestAnimationFrame, this is the least we need. We require the style recalculation to be done.
the fifth is the setTimeout, coming less than a milisecond later. But being in it's own task.

Reply:
Definitely prefer the requestAnimationFrame then, cool stuff 👍